### ISSUE TITLE
SyntaxError: Unexpected token u in JSON at position 0

### ISSUE THREAD

**Issue by crackernutter** [Esri/cedar#405]
I  copied the code from the current GitHub ReadMe (it's the only code in my js file), and the only element on my page is a div for the chart.  I figured this is the simplest way to start possible, but I'm still receiving an error....

![image](https://user-images.githubusercontent.com/6819550/37302794-efc1825a-2602-11e8-9cb0-ffd36e672490.png)

Any ideas here?  I haven't dug into the actual FeatureServer endpoint, but a standard 1=1 query returns valid json.

**Comment by tomwayson** [comment]
I apologize @crackernutter 

It looks like there were a couple of bad lines in there. I'm updating the README now, but for reference, this should work:

```html
<html>
  <head>

  </head>
  <body>
    <div id="chart" style="height: 400px;"></div>
    <!-- load the amCharts base library -->
    <script src="https://www.amcharts.com/lib/3/amcharts.js"></script>
    <!-- in this case, we only need bar charts, so we'll load the appropriate amCharts script -->
    <script src="https://www.amcharts.com/lib/3/serial.js"></script>
    <!-- optionally load an amcharts theme; cedar provides a calcite theme -->
    <script src="https://unpkg.com/@esri/cedar/dist/umd/themes/amCharts/calcite.js"></script>
    <!-- load cedar -->
    <script src="https://unpkg.com/@esri/cedar/dist/umd/cedar.js"></script>
    <script>
      // connect to the data
      var datasets = [{
        "url": "https://services.arcgis.com/uDTUpUPbk8X8mXwl/arcgis/rest/services/Public_Schools_in_Onondaga_County/FeatureServer/0",
        "name": "schools",
        "query": {
          "orderByFields": "Number_of_SUM DESC",
          "groupByFieldsForStatistics": "Type",
          "outStatistics": [{
            "statisticType": "sum",
            "onStatisticField": "Number_of",
            "outStatisticFieldName": "Number_of_SUM"
          }]
        }
      }];

      // designate a one or more series to show the data on the chart
      var series = [{
        "category": {"field": "Type", "label": "Type"},
        "value": {"field": "Number_of_SUM", "label": "Number of Students"},
        "source": "schools"
      }];

      // optinally override any of the cart type's default styles
      var overrides = {
        "categoryAxis": {
          "labelRotation": -45
        }
      }

      //create a cedar chart using the known 'bar' type
      var elementId = 'chart';
      var chart = new cedar.Chart(elementId, {"type": "bar"})
        .datasets(datasets)
        .series(series)
        .overrides(overrides);

      // render the chart
      chart.show();
    </script>
  </body>
</html>
```

**Comment by tomwayson** [comment]
resolved in fe6a99ef32fe5a6463540bf088af4a431df52015
